auth-web/src/routes/router.ts

```diff
@@ -221,78 +218,24 @@ export function getRoutes (): RouteConfig[] {
       path: '/business',
       name: 'business-root',
       meta: { requiresAuth: true, showNavBar: true },
-      redirect: `/account/${JSON.parse(ConfigHelper.getFromSession(SessionStorageKeys.CurrentAccount) || '{}').id || 0}/business`
+      beforeEnter: () => {
+        // Get the account ID
+        const orgId = JSON.parse(ConfigHelper.getFromSession(SessionStorageKeys.CurrentAccount) ?? '{}').id ?? 0
+        // Use the redirect function with a mock route object
+        return redirectToBusinessDashboard({ params: { orgId }, query: {} })
+      }
     },
     {
       path: '/account/:orgId',
       name: 'account',
-      beforeEnter: (to, from, next) => {
-        // redirect to new business registry dashboard
-        if (LaunchDarklyService.getFlag(LDFlags.EnableBusinessRegistryDashboard) && !to.query.noRedirect) {
```

Comment:
Please add a comment to the Auth Web release ticket to archive this FF after deployment to Prod.